Gemfile

```diff
@@ -14,7 +14,7 @@ chef_vault_version = '> 3.0'
 chef_version = if Bundler.current_ruby.on_23?
                  '= 12.18.31'
                else
-                 '= 14.13.11'
```

Comment:
I think you need another else if case in here to continue using Chef 14 when the current ruby is 2.5.